### ISSUE TITLE
Handling missing ~/.nuget/packages directory

### ISSUE THREAD

**Issue by jthelin** [terrajobst/dotnet-nuget-gc#12]
An ugly exception (see below) is thrown if the `~/.nuget/packages` directory is missing.

FWIW, I suspect I may have deleted that directory on this system during a manual disk cleanup / space reclamation binge before I found out about `nuget-gc` ;)

# Repro steps:

1. `dotnet tool install dotnet-nuget-gc -g`
2. `rm -fR ~/.nuget/packages`
3. `dotnet nuget-gs`

# Failure mode:

```
$ dotnet nuget-gc

Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path '/home/MYUSER/.nuget/packages'.
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)
   at System.IO.Enumeration.FileSystemEnumerator`1..ctor(String directory, EnumerationOptions options)
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options)
   at System.IO.Enumeration.FileSystemEnumerableFactory.DirectoryInfos(String directory, String expression, EnumerationOptions options)
   at System.IO.DirectoryInfo.InternalEnumerateInfos(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)
   at System.IO.DirectoryInfo.GetDirectories(String searchPattern, EnumerationOptions enumerationOptions)
   at System.IO.DirectoryInfo.GetDirectories()
   at NugetCacheCleaner.Program.CleanCache(Boolean force, TimeSpan minDays) in /home/vsts/work/1/s/src/Program.cs:line 75
   at NugetCacheCleaner.Program.Main(String[] args) in /home/vsts/work/1/s/src/Program.cs:line 40
```

Possibly related issues: #7

**Comment by jthelin** [comment]
I submitted PR #13 to address this issue.

**Comment by sungam3r** [comment]
This repo seems to be abandoned. Check out the commit history.

**Comment by terrajobst** [comment]
Fixed by #13
